indico/modules/events/editing/client/js/components/Timeline/judgment/AcceptRejectForm.jsx

```diff
@@ -0,0 +1,61 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import React from 'react';
+import {useDispatch, useSelector} from 'react-redux';
+import PropTypes from 'prop-types';
+import {Form as FinalForm} from 'react-final-form';
+import {Form} from 'semantic-ui-react';
+
+import {FinalSubmitButton, FinalTextArea} from 'indico/react/forms';
+import {Translate} from 'indico/react/i18n';
+
+import {reviewEditable} from '../../../actions';
+import * as selectors from '../../../selectors';
+import {blockPropTypes} from '../util';
+
+import './JudgmentBox.module.scss';
+
+export default function AcceptRejectForm({block, action, setLoading}) {
+  const {eventId, contributionId, editableType} = useSelector(selectors.getStaticData);
+  const dispatch = useDispatch();
+
+  return (
+    <FinalForm
+      onSubmit={async formData => {
+        setLoading(true);
+        await dispatch(
```

Comment:
if you use submitFormAction in the action you can use the return value of the dispatch to pass error information from the server back to final-form (and thus show errors related to a specific field next to that field as long as the field names match on client+server)